iOS/Core/ios-config.json

```diff
@@ -6432,10 +6432,6 @@
                 "allowPurchase": {
                     "state": "enabled"
                 },
-                "useUnifiedFeedback": {
-                    "state": "enabled",
-                    "minSupportedVersion": "7.136.0"
-                },
```

Comment:
This will get re-added by the automatic embedded file update process, but that'll be fine. We'll have to leave it in the config anyway so that old versions keep using the form.

Reply:
Actually, we will need to undo this change anyway, since we have unit tests for this file compared to its SHA. Let me do that now.